molgenis/molgenis-emx2
# fix: allow default values of refs to change before save

how to test:
- set default value on a ref column e.g. category in the pet table of the pet story to `={name:"cat"}`
- Create a new record. See the default value in place already.
- Try to change the set default
- Save

todo:
